### DISCUSSION TITLE
Signpost should remain as signpost if the fit and not be modal panels

### DISCUSSION THREAD

**Discussion by rbirkgit** [vmware-clarity/core#63] [General]
The new Core signpost switches behavior into modal panels if the signpost is bigger that the signpost's container. This is a common scenario. I find this behavior odd and have not seen anything like this before. 

Can we change the core signposts to behave more like regular signposts?

**Comment by colinreedmiller** [comment]
I don't know exactly why this has been implemented as it has but my suspicion is that it has to do with responsiveness. How components behave at different viewports is an important consideration for Core. I suspect a flag to suppress this behavior could be added when a (desktop) application prefers it. We would need to go into the signpost code and see if we can answer these questions since what I have included here is largely speculation in the absence of dev notes from the component design phase.
